### PR TITLE
PLDM support power caps based on power supply number,type,input voltage

### PR DESCRIPTION
**Pull request by baileysh**
RTC: 209572
Change-Id: I4bb8985c43f54dadf85a92c96cc81a402cfaa4a5
Change-Id: I073ed2ac00a21fe4e204bd05235f173183a9b719

### CODE REVIEW

File: oem/ibm/configurations/bios/enum_attrs.json [hunk 1]
```diff
@@ -451,48 +451,60 @@
             "readOnly": true
         },
         {
-            "attribute_name": "pvm_keep_and_clear",
-            "possible_values": ["Disabled", "Enabled"],
+            "attribute_name": "hb_power_PS0_functional",
```
Comment (Pavithrab7) [inline oem/ibm/configurations/bios/enum_attrs.json:454]: Why is the pvm_keep_and_clear attribute modified?

File: oem/ibm/configurations/bios/enum_attrs.json [hunk 2]
```diff
@@ -451,48 +451,60 @@
...
+                "interface": "xyz.openbmc_project.State.Decorator.OperationalStatus",
+                "property_name": "Functional",
+                "property_type": "bool",
+                "property_values": [true, false]
+            }
         },
         {
-            "attribute_name": "pvm_clear_nvram",
-            "possible_values": ["Disabled", "Enabled"],
+            "attribute_name": "hb_power_PS1_functional",
```
Comment (Pavithrab7) [inline oem/ibm/configurations/bios/enum_attrs.json:468]: Same as above.
Shouldn't these attributes be added at the end and not modify existing attributes?

File: oem/ibm/configurations/bios/enum_attrs.json (hunk 1 above)
Comment (baileysh) [reply oem/ibm/configurations/bios/enum_attrs.json:454]: It is not being modified,, it is in a higher part of the file. see line 410 above.
This has something to do with the tools,

File: oem/ibm/configurations/bios/enum_attrs.json (hunk 2 above)
Comment (baileysh) [reply oem/ibm/configurations/bios/enum_attrs.json:468]: See line 410 above.  This has something to do with what the tool thinks is happening.

File: oem/ibm/configurations/bios/enum_attrs.json (hunk 1 above)
Comment (Pavithrab7) [reply oem/ibm/configurations/bios/enum_attrs.json:454]: yes, looks like there were duplicate entries for the same attribute.

**Review by Pavithrab7 — APPROVED** [review]

File: oem/ibm/configurations/bios/enum_attrs.json (hunk 2 above)
Comment (Pavithrab7) [reply oem/ibm/configurations/bios/enum_attrs.json:468]: yes, looks like there were some duplicate entries

**Review by jaypadath — APPROVED** [review]

**Review by Pavithrab7 — APPROVED** [review]

**Pull request merged**